modules/fts/pages/fts-custom-filters.adoc

```diff
@@ -52,95 +51,95 @@ image::fts-custom-filters-panel-new-character-filter.png[,700,align=left]
 Adds a new tokenizer to the list of those available.
 
 The new tokenizer becomes available for inclusion in custom-created analyzers.
-** Left-click the *+ Add Tokenizer* 
 
-It displays the *Custom Tokenizer* dialog:
+Left-click the *+ Add Tokenizer*. It displays the *Custom Tokenizer* dialog:
 
 [#fts_custom_filters_tokenizer_dialog_initial]
 image::fts-custom-filters-tokenizer-dialog-initial.png[,380,align=left]
-+
+
 The following interactive fields are provided:
 
- ** *Name*: A suitable, user-defined name for the new tokenizer.
- ** *Type*: The process used in tokenizing.
-Available options can be accessed from the pull-down menu at the right of the field.
+* *Name*: A suitable, user-defined name for the new tokenizer.
+
+* *Type*: The process used in tokenizing. Available options can be accessed from the pull-down menu at the right of the field.
 (Currently, `regexp` and `exception` are available.)
- ** *Regular Expression*: The specific _regular expression_ used by the tokenizing process.
-+
+
+* *Regular Expression*: The specific _regular expression_ used by the tokenizing process.
+
 The following completed fields define a tokenizer that removes uppercase characters:
-+
+
 [#fts_custom_filters_tokenizer_dialog_completed]
 image::fts-custom-filters-tokenizer-dialog-completed.png[,380,align=left]
-+
+
 When saved, the new tokenizer is displayed on its own row, with options for further editing and deleting:
-+
+
 [#fts_custom_filters_panel_new_tokenizer]
 image::fts-custom-filters-panel-new-tokenizer.png[,700,align=left]
 
 === Token filter
- Adds a new token filter to the list of those available.
-The new token filter becomes available for inclusion in custom-created analyzers.
-** Left-click the [.ui]*+ Add Token Filter* 
 
-It displays the *Custom Token Filter* dialog:
+Adds a new token filter to the list of those available. The new token filter becomes available for inclusion in custom-created analyzers.
+
+Left-click the [.ui]*+ Add Token Filter*. It displays the *Custom Token Filter* dialog:
```

Comment:
The [.ui] role hasn't been used in other places that are doing the same thing.

Reply:
Updated